x-pack/plugin/esql/qa/testFixtures/src/main/resources/spatial.csv-spec

```diff
@@ -2,25 +2,6 @@
 # Tests for GEO_POINT type
 #
 
-# This first test is only here to verify type support in older versions, all other tests are for newer versions
```

Comment:
We could create another test that works in older versions, like parsing WKT and then printing WKT, picking a value that matches 8.12 quantization so we get the same values in both 8.12 and 8.13.

Reply:
added